### DISCUSSION TITLE
Comments section: Move `Translate to *language name*` to comment option menu

### DISCUSSION THREAD

**Discussion by HarGabt** [ReVanced/revanced-discussions#177] [Question]
Basically, this to avoid unnecessary translation of a comment and making a user to push it again to bring the original comment back.

![Screenshot_20220507_123351_com vanced android youtube](https://user-images.githubusercontent.com/44652400/167246619-7b7efaea-39af-4d6f-8feb-a7cd46a1377c.jpg)

**Comment by NebelNidas** [comment]
Though it should be optional and off by default, since most people only speak ~two languages and you quite often stumble upon foreign comments
